index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <title>My Portfolio</title>
+  </head>
+  <body>
+    <div>
+      <nav class="navbar">
+        <img src="/assets/My Logo.svg" class="logo" />
+        <img class="menu" src="assets/Side.svg" alt="menu" />
+      </nav>
+      <section class="my-section">
+        <div>
+          <h1 class="intro">I'm Walid <br />Glad to see <br />You!</h1>
+        </div>
+        <div>
+          <p class="about-me">
+            I’m a software developer! I can help you <br />
+            build a product , feature or website Look <br />
+            through some of my work and <br />
+            experience! If you like what you see and <br />
+            have a project you need coded, don’t <br />
+            hestiate to contact me.
+          </p>
+        </div>
+        <div class="contact-me">
+          <h2 class="connect">LET'S CONNECT</h2>
+        </div>
+        <div class="buttons">
+          <a href=""><img src="/assets/Github.svg" alt="Github" /></a>
+          <a href=""><img src="/assets/Twitter.svg" alt="Twitter" /></a>
+          <a href=""><img src="/assets/medium.svg" alt="Medium" /></a>
+          <a href=""><img src="/assets/Vector.svg" alt="" /></a>
+        </div>
```

Comment:
- [ ] Please these icons form a list and should be wrapped in a `ul` tag with `li` elements this is semantically better.